### ISSUE TITLE
GUI: number of splitting covariates

### ISSUE THREAD

**Issue by ddalthorp** [ddalthorp/GenEst#520]
If the number of splitting is covariates is set too high, the error is flagged appropriately. But if the error is then fixed and "Split Estimate" is again clicked, then gray screen.

**Comment by juniperlsimonis** [comment]
hmmm, I'm not able to re-create this with the mock data set, can you walk me through the steps you took to get this error?

**Comment by ddalthorp** [comment]
start with 1 SS split + 2 CO splits --> error message
Then, erase the SS split --> gray screen

**Comment by juniperlsimonis** [comment]
right, that's what i've tried and i can't re-create that error. what data set did you use, what inputs, and what models did you select?
